src/routes/contracts/__tests__/get-contract.e2e-spec.ts

```diff
@@ -29,14 +27,85 @@ describe('Get contract e2e test', () => {
 
   it('GET /contracts/<address>', async () => {
     const contractAddress = '0x7cbB62EaA69F79e6873cD1ecB2392971036cFAa4';
-    const expectedResponse: Contract = JSON.parse(
-      readFileSync(
-        'src/routes/contracts/__tests__/resources/contract-expected-response.json',
-        {
-          encoding: 'utf-8',
-        },
-      ),
-    );
+    const expectedResponse = {
+      address: '0x7cbB62EaA69F79e6873cD1ecB2392971036cFAa4',
+      name: 'CreateCall',
+      displayName: '',
+      logoUri: null,
+      contractAbi: {
+        abi: [
+          {
+            name: 'ContractCreation',
+            type: 'event',
+            inputs: [
+              {
+                name: 'newContract',
+                type: 'address',
+                indexed: false,
+                internalType: 'address',
+              },
+            ],
+            anonymous: false,
+          },
+          {
+            name: 'performCreate',
+            type: 'function',
+            inputs: [
+              {
+                name: 'value',
+                type: 'uint256',
+                internalType: 'uint256',
+              },
+              {
+                name: 'deploymentData',
+                type: 'bytes',
+                internalType: 'bytes',
+              },
+            ],
+            outputs: [
+              {
+                name: 'newContract',
+                type: 'address',
+                internalType: 'address',
+              },
+            ],
+            stateMutability: 'nonpayable',
+          },
+          {
+            name: 'performCreate2',
+            type: 'function',
+            inputs: [
+              {
+                name: 'value',
+                type: 'uint256',
+                internalType: 'uint256',
+              },
+              {
+                name: 'deploymentData',
+                type: 'bytes',
+                internalType: 'bytes',
+              },
+              {
+                name: 'salt',
+                type: 'bytes32',
+                internalType: 'bytes32',
+              },
+            ],
+            outputs: [
+              {
+                name: 'newContract',
+                type: 'address',
+                internalType: 'address',
+              },
+            ],
+            stateMutability: 'nonpayable',
+          },
+        ],
+        description: 'CreateCall',
+        relevance: 100,
+      },
+      trustedForDelegateCall: false,
+    };
```

Comment:
No strong opinion on this one but having the JSON file here also makes sense. The `IBuilder` is more to set specific values while not having the need to set values for every property.

Reply:
Yes, totally agree, but I didn't use a builder-generated instance here, I just pasted the JSON payload for the expected response into the test file. This approach is used in other parts of the code, and there are no more JSON files now, so I leaning towards using the payload directly and avoiding reading external JSON files. But no strong opinion either, let me know if you prefer to use a JSON file here 🙂

Reply:
> Yes, totally agree, but I didn't use a builder-generated instance here, I just pasted the JSON payload for the expected response into the test file. This approach is used in other parts of the code, and there are no more JSON files now, so I leaning towards using the payload directly and avoiding reading external JSON files. But no strong opinion either, let me know if you prefer to use a JSON file here 🙂

Yes but since the PR was about replacing instances with an `IBuilder` and this wouldn't be replaced by one I was wondering if it'd make sense to drop the JSON files here too.

Reply:
Yes, that's a totally fair point. I was removing all the test data JSON files from the project but maybe this one was slightly out of the scope of the changes. If you agree I'd opt for keeping the payload within the file but no strong opinion either 🙂 (we can take the JSON back in another PR, or even try to find a simpler ABI so the test data gets less verbose).